search-service/src/main/java/nl/aerius/search/tasks/async/SearchResult.java

```diff
@@ -50,6 +50,7 @@ public void failureComplete() {
     if (LOG.isTraceEnabled()) {
       LOG.error("Search task {} has completed with a failure.", uuid);
     }
+    results.clear();
```

Comment:
Personally I'd be fine if the search request returned a failure because 1 of the services failed, but also some usable results